src/workloads/execution/MultiPlanning.yml

```diff
@@ -0,0 +1,148 @@
+SchemaVersion: 2018-07-01
```

Comment:
Just a heads up: once this is done, it's worth incorporating into the SBE perf dashboard. It's a slightly annoying process right now, unfortunately. You have to add the task name here: https://github.com/10gen/PerformanceReports/blob/main/scripts/perf_reports_sbe_perf.yml

Then once the measurements show up as untagged in the dashboard, you have to add the necessary tags to this file: https://github.com/10gen/PerformanceReports/blob/main/scripts/tags.yml

Maybe one day if we something like our PerformanceReports system becomes a more official tool owned by dev prod / Evergreen there can be some work done to make this whole process of incorporating a new benchmark into the dashboard less painful.

@mtandrei I assume you would agree that these benchmarks should be incorporated into the dashboard?

Reply:
I will do it after the merge.

Reply:
Sounds good. Leaving this comment thread open now in case Mihai wants to jump in.